app/services/symbolService.js

```diff
@@ -119,22 +119,23 @@ class SymbolService {
       const ohlcv = await exchange.fetchOHLCV(symbol, interval, currentTimestamp, limit);
       // Adjust currentTimestamp for the next day (8 AM)
       currentTimestamp += 24 * 60 * 60 * 1000;
-      allOHLCV = allOHLCV.concat(ohlcv);
+      allOHLCV.push(...ohlcv);
     }
     // Process the price data
-    await this.processPriceData(allOHLCV, exchangeId, symbol, otherCurrency, exchangeUrl);
+    await this.savePriceData(allOHLCV, exchangeId, symbol, otherCurrency, exchangeUrl);
 
     return allOHLCV;
   }
 
-  async processPriceData(allOHLCV, exchange, symbol, otherCurrency, exchangeUrl){
+  // save the data we obtain from ccxt to database for future use
+  async savePriceData(allOHLCV, exchange, symbol, otherCurrency, exchangeUrl){
```

Comment:
It'll be better if the function name is 'updateSymbol', and go with PATCH in route.